b2/account_info.py

```diff
@@ -141,6 +141,42 @@ def clear_large_file_upload_urls(self, file_id):
         pass
 
 
+class UploadUrlPool(object):
```

Comment:
I noticed that a unit test for this class is missing, but this is a very simple class and I am fine with not having a test for it. It will be integration-tested anyway.